_lexicon/world-state.md

```diff
@@ -1,5 +1,7 @@
 ---
 layout: post
 title: World State
-description:  Given that the __Information__ in an __Environment__ may change with respect to some parameter (for example, time), __World State__ defines a static snapshot of the __Information__ within an __Environment__. In other words, a __World State__ is an __Environment__ where all __Information__ is held constant. Denoted $$ w \in W $$.
----
\ No newline at end of file
+description:  Given that the __Information__ in an __Environment__ may change with respect to some parameter (for example, time), __World State__ defines a static snapshot of the __Information__ within an __Environment__. In other words, a __World State__ is an __Environment__ where all __Information__ is held constant. Denoted $$ w \in W $$. When used with a reference Agent, $$ w_A \in W_A $$, __World State__ refers to a static snapshot of an Agent's __World Model__ as opposed to the __World__ itself.
```

Comment:
I think we'll have to revisit whether the World has a state independent of any agents. (Hoffman, QBism, etc)